testsuite/pyunit/lsp/LanguageServer.py

```diff
@@ -197,6 +199,7 @@ def test_Request_Response(self):
 class Test004_Error_Project(JSONTest):
     subdir = Path("004errprj")
 
+    @mark.xfail
```

Comment:
Why ?  lsp doesn't depend on dom.

Reply:
This is the test that fails since long time only Windows.
You couldn't fix/reproduce it on you side.

It's about the JSON compares with `/` vs. `\`.

Reply:
```
XFAIL [ 85%]
Test004_Error_Project
[0].params.message: mismatch (ref: json error in project file C:\git\GitHub\GHDL\ghdl\testsuite\pyunit\lsp/004errprj/hdl-prj.json:1:3, result: json error in project file C:\git\GitHub\GHDL\ghdl\testsuite\pyunit\lsp\004errprj\hdl-prj.json:1:3)
FAILURE between output and C:\git\GitHub\GHDL\ghdl\testsuite\pyunit\lsp\004errprj\replies.json (for C:\git\GitHub\GHDL\ghdl\testsuite\pyunit\lsp\004errprj\cmds.json)
Writing result output to result.json
```

Reply:
Is it possible to mark it xfail only on windows platforms ?

Reply:
I'm not aware of a method to make `xfail` platform specific. I have reverted the change and pushed again.

This should finish in 20 minutes. I needed to force push again due to black complaints.

Reply:
We can xfail or skip specific tests on Windows or on MSYS2 only:

- https://github.com/ghdl/ghdl-cosim/blob/419bd73fd5ff8e22faa430925a994dae96cbcc0d/tests/common.py#L13
- https://github.com/ghdl/ghdl-cosim/blob/master/tests/test_vhpidirect_vffi.py#L16-L21
- https://github.com/ghdl/ghdl-cosim/blob/master/tests/test_vhpidirect_shared.py#L26-L29